server/migrations/20210925191009-create-post.ts

```diff
@@ -0,0 +1,44 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface: any, Sequelize: any) => {
+    await queryInterface.createTable("Posts", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.STRING,
+      },
+      title: {
+        type: Sequelize.TEXT,
+      },
+      body: {
+        type: Sequelize.STRING,
+      },
+      thumbnail: {
+        type: Sequelize.STRING,
+      },
+      tags: {
+        type: Sequelize.STRING,
+      },
+      location: {
+        type: Sequelize.STRING,
+      },
+      capacity: {
+        type: Sequelize.INTEGER,
+      },
+      feedbackScore: {
```

Comment:
feedbackScore?

Reply:
A posts score is updated based on interactions, and if it's reported. We want to automatically remove posts that are often reported, so we can have a score that goes down on reports.